app/controllers/hyku/invitations_controller.rb

```diff
@@ -11,6 +11,7 @@ def after_invite_path_for(_resource)
     # override the standard invite so that accounts are added properly
     # if they already exist on another tenant and invited if they do not
     def create
+      authorize! :grant_admin_role, User if params[:user][:roles] == ::RolesService::ADMIN_ROLE
```

Comment:
is params[:user][:roles] singular or plural? `roles` implies plural. maybe:

```suggestion
      authorize! :grant_admin_role, User if Array.wrap(params[:user][:roles]).include?(::RolesService::ADMIN_ROLE)
```

Reply:
Users used to be able to be invited with more than one role, but this work changes it to just one at a time (using a dropdown). I didn't update the plurality when I made that change, so I just did [in this commit](https://github.com/samvera/hyku/pull/1859/commits/be42b65f4a49501fb2c5d713562706a8f5b8e88e)